src/style.css

```diff
@@ -1,3 +1,74 @@
+:root {
+  font-family: Roboto, serif;
+  background-color: aqua;
+  margin: 0;
+  padding: 0;
+
+  --main-color: #00ACDC;
+}
+
 body {
   margin: 0;
 }
+
+.header {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  background-color: #fff;
+  height: 60px;
+  padding: 0 50px;
+}
+
+ul.nav_list li {
+  display: inline-flex;
+}
+
+.nav_list {
+  line-height: 60px;
+  font-weight: bold;
+  font-size: 12px;
+  text-transform: uppercase;
+  list-style: none;
+  margin: 0;
+  padding: 0;
+}
+
+.nav__link {
+  color: black;
+  height: 60px;
```

Comment:
it's no a good practice to hardcode height for elements with content inside. use padding instead